export function createErrorString({
  message,
  file,
  line,
  column,
}: {
  message: string;
  file: string;
  line: number;
  column: number;
}) {
  return `
// error generated by vite-plugin-vitest-typescript-assert
(() => {
  const err = new TypeError("${message}");
  err.name = "TypeError";
  err.stack = "";
  err.stackStr = "";
  err.stacks = [{
    file: "${file}",
    line: ${line + 1},
    column: ${column},
    sourcePos: {
      source: "${file}",
      line: ${line + 1},
      column: ${column}
    }
  }];
  throw err;
})()
`;
}
